README.md

```diff
@@ -27,10 +27,14 @@ The first two items are provided by the `influxdb::install` class and are restri
 
 InfluxDB resources are managed by the various types and providers and have a dependency on the `influxdb` class.  This is because we need to be able to enumerate and query resources on either a local or remote machine, and the `influxdb` class provides a kind of "base" resource for the types and providers to inherit.
 
-When managing InfluxDB resources in Puppet code, you will need to include this class as a dependency, for example by including a `require`in your manifest.
+When managing InfluxDB resources in Puppet code, you will need to include this class as a dependency by inheriting from this class in your manifest.  For example:
 
 ```
-require influxdb
+class my_profile::my_class(
+  String $my_param,
+) inherits influxdb {
```

Comment:
Class inheritance is a bad pattern in Puppet. It should only be used for the `params.pp` and that is a deprecated format. From https://puppet.com/docs/puppet/7/style_guide.html#style_guide_classes-class-inheritance

> Don't use class inheritance; use data binding instead of params.pp pattern. Inheritance is used only for params.pp, which is not recommended in Puppet 4.

Reply:
The whole point of this was to remove that, and... I forgot to.  I removed the base type/provider, but not the manifest.  I'll remove this and update the readme.